src/components/NavBar/NavBar.js

```diff
@@ -21,17 +21,24 @@ const NavBar = () => {
                         alt="acm logo"
                         onClick={() => navigate("/")} />
                 </div>
-                <div className="element-container">
+                
+                <div className="element-container resouce-btn">
```

Comment:
really minor but i think there's a typo for the class name lol should be `resource-btn` right? i see that the same typo is in the `NavBar.css` file